Lesson19/Ece/script.js

```diff
@@ -19,35 +18,62 @@
 
 */
 // 1. Change the text color of all paragraphs on the page to #272932.
-var paragraphs = document.querySelectorAll('p');
-console.log('Excercise 1', paragraphs);
+var paragraphs = document.querySelectorAll("p");
+console.log("Excercise 1", paragraphs);
 for (var i = 0; i < paragraphs.length; i++) {
-  paragraphs[i].style.color = 'red';
+  paragraphs[i].style.color = "red";
 }
 // 2. Change the background color of the entire page to #eff1f3.
-const mainElement = document.getElementById('main');
-console.log('Excercise 2', mainElement);
-mainElement.style.backgroundColor = 'grey';
+const mainElement = document.getElementById("main");
+console.log("Excercise 2", mainElement);
+mainElement.style.backgroundColor = "grey";
 // 3. Add a new paragraph to the bottom of the page with this text: "Lorem ipsum dolor sit amet consectetur adipisicing elit. Illum esse, recusandae sint error possimus hic, voluptates vel eveniet est, deleniti reprehenderit fugit minus? Officiis iusto, incidunt molestiae neque quas omnis."
 const newParagraph = document.createElement("p");
-newParagraph.innerHTML="Lorem ipsum dolor <a href=\"https://dsds.dsds\">sit amet consectetur </a>adipisicing elit. Illum esse, recusandae sint error possimus hic, voluptates vel eveniet est, deleniti reprehenderit fugit minus? Officiis iusto, incidunt molestiae neque quas omnis.";
+newParagraph.innerHTML =
+  'Lorem ipsum dolor <a href="https://dsds.dsds">sit amet consectetur </a>adipisicing elit. Illum esse, recusandae sint error possimus hic, voluptates vel eveniet est, deleniti reprehenderit fugit minus? Officiis iusto, incidunt molestiae neque quas omnis.';
 document.body.appendChild(newParagraph);
 
-const firstParagraph = document.getElementById('first-paragraph');
+const firstParagraph = document.getElementById("first-paragraph");
 // If it was a list, you could use the code below to insert the new element
 // firstParagraph.insertBefore(newParagraph, positionWhereToInsert);
 
 // 4. Remove the second image from the page.
-const images = document.body.querySelectorAll('img');
+const images = document.body.querySelectorAll("img");
 console.log(images);
 images[1].remove();
 
 // 5. Add a border to all images on the page (2px solid #272932).
+const allImages = document.body.querySelectorAll("img");
+console.log("allImages");
+allImages.style.border = "2px solid #272932";
```

Comment:
This line does not work as the type of `allImages` is a node list. 
Whenever you select multiple elements, you need to iterate through them and do the operation for each element of the list.
```suggestion
allImages.forEach(image => image.style.border = "2px solid #272932");

```